### PR TITLE
README: Adds NuGet badges

### PR DESCRIPTION
**Pull request by abatishchev**
# Pull Request Template

## Description

Replaced useless ADO pipeline badge "never built" with working badges for NuGet packages.

Note: shield.io is a 3rd party website but is considered one of the most popular across OSS

## Type of change

Please delete options that are not relevant.

- [x] Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://cosmos-db-sdk-public.visualstudio.com/cosmos-db-sdk-public/_apis/build/status/azure-cosmos-dotnet-v3?branchName=master)](https://cosmos-db-sdk-public.visualstudio.com/cosmos-db-sdk-public/_build/latest?definitionId=41&branchName=master)
+[![NuGet](https://img.shields.io/nuget/v/Microsoft.Azure.Cosmos.svg)](https://www.nuget.org/packages/Microsoft.Azure.Cosmos)
```
Comment (j82w) [inline README.md:1]: Is there any reason to not just put the svg in the github repo?

Comment (abatishchev) [reply README.md:1]: The svg is generated dynamically based on the latest version of the corresponding NuGet package. That's the point of a badge :)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+[![NuGet Prerelease](https://img.shields.io/nuget/vpre/Microsoft.Azure.Cosmos.svg)](https://www.nuget.org/packages/Microsoft.Azure.Cosmos)
```
Comment (ealsur) [inline README.md:2]: Could you add a screencapture on the Description of what it shows when the markdown is rendered?

Comment (abatishchev) [reply README.md:2]: Sure, you can see the rendered badges here: https://github.com/Azure/azure-cosmos-dotnet-v3/blob/7955de1e0bae8e96a07c210f4c2914112523fd6a/README.md
i.e. Files changed -> View file

Comment (abatishchev) [reply README.md:2]: @j82w what you think?

**Comment by abatishchev** [comment]
@j82w, @ealsur ping on this, please.

File: README.md (hunk 2 above)
Comment (j82w) [reply README.md:2]: @kirankumarkolli is this ok to merge or is there some internal Microsoft approval process we need to go through first? I did find https://github.com/microsoft/ReportingServicesTools is using it.

Comment (abatishchev) [reply README.md:2]: @j82w, @kirankumarkolli it's super widely used, searching for `img.shields.io` in the microsoft org [yields 681 results](https://github.com/search?q=org%3Amicrosoft+img.shields.io&type=code).

**Comment by abatishchev** [comment]
@kirankumarkolli ping, please.

**Review by j82w — APPROVED** [review]

**Comment by j82w** [comment]
Cosmos Node JS github repo already uses it. https://github.com/Azure/azure-sdk-for-js/edit/master/sdk/cosmosdb/cosmos/README.md Just waiting for the gates to finish. It hit a transient test failure.

**Pull request merged**